internal/version/version.go

```diff
@@ -0,0 +1,23 @@
+package version
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"time"
+)
+
+var Version string = "0.0.0"
+var GitCommit string = "unknown"
+var BuildTime string = "unknown"
```

Comment:
Rewrite for clarity:

    // These vars get bound at build time using the --ldflags mechanism.
    var (
            Version   = "0.0.0"
            GitCommit = "unknown"
            BuildTime = "unknown"
    )